### PR TITLE
Add: core_rc

### PR DESCRIPTION
**Pull request by Likyaz**
Add Core: 

- Init database and feed with basic data
- Cmd to generator development fixture
- Skin service and provider
- DeepL service
- S3Destination

### CODE REVIEW

**Review by Cod3lta — APPROVED** [review]

File: core_rc/models/Donation.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class Donation(models.Model):
+    DONATION_PROCESSORS = (
+        ('pp', 'paypal'),
+    )
+
+    PROVIDER_PROCESSORS = (
+        ('mc', 'minecraft'),
```
Comment (lululombard) [inline core_rc/models/Donation.py:10]: might need changes

File: core_rc/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+from core_rc.models.Donation import Donation
+
+from core_rc.models.article.Article import Article
+from core_rc.models.article.Category import Category
+from core_rc.models.article.Post import Post
+from core_rc.models.article.LocalizedArticle import LocalizedArticle
+from core_rc.models.article.LocalizedCategory import LocalizedCategory
+from core_rc.models.article.LocalizedPost import LocalizedPost
+
+from core_rc.models.url.ReducedUrl import ReducedUrl
```
Comment (lululombard) [inline core_rc/models/__init__.py:13]: old name

File: core_rc/models/url/ReducedUrl.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class ReducedUrl(models.Model):
```
Comment (lululombard) [inline core_rc/models/url/ReducedUrl.py:4]: old name

File: core_rc/models/url/ReducedUrl.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    url_token = models.ForeignKey(
+        'UrlToken',
+        on_delete=models.CASCADE,
+    )
+    url = models.CharField(max_length=2048)
+    shortened = models.CharField(max_length=32)
+
+    def __str__(self):
+        return f"<ReducedUrl: {self.shortened} - {self.url_token.access_name}>"
```
Comment (lululombard) [inline core_rc/models/url/ReducedUrl.py:13]: old name

File: core_rc/service/SkinService.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+
+class SkinService:
+    class RendererType(Enum):
+        BODY = auto()
+        TEMPLATE = auto()
+        HEAD = auto()
+
+    @staticmethod
+    def __getBody(img_template, slim):
```
Comment (lululombard) [inline core_rc/service/SkinService.py:15]: should be snake_case

**Review by lululombard — CHANGES_REQUESTED** [review]
Some changes required

File: core_rc/models/InfoProviderPlayer.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+
+class InfoProviderPlayer(models.Model):
```
Comment (lululombard) [inline core_rc/models/InfoProviderPlayer.py:4]: PlayInfoProvider maybe?

File: core_rc/migrations/0001_initial.py [hunk 7]
```diff
@@ -68,7 +77,7 @@ class Migration(migrations.Migration):
             ],
         ),
         migrations.CreateModel(
-            name='ReducedUrl',
+            name='UrlShortener',
```
Comment (lululombard) [inline core_rc/migrations/0001_initial.py:80]: ShortUrl

**Review by lululombard — CHANGES_REQUESTED** [review]

**Review by lululombard — APPROVED** [review]

**Pull request merged**